docs/pages/kubernetes-access/guides.mdx

```diff
@@ -4,7 +4,15 @@ description: Detailed guides for configuring Teleport Kubernetes Access.
 layout: tocless-doc
 ---
 
-- [Teleport Kubernetes Access for CI/CD](./guides/cicd.mdx): Teleport Kubernetes Access for CI/CD.
-- [Teleport Kubernetes Access and Trusted Clusters](./guides/federation.mdx): Federated Access using Teleport Trusted Clusters.
-- [Kubernetes Access Multiple Clusters](./guides/multiple-clusters.mdx): Connecting Multiple Clusters with Teleport Kubernetes Access.
-- [Kubernetes Access from standalone Teleport](./guides/standalone-teleport.mdx): Connecting standalone Teleport installations to Kubernetes clusters
\ No newline at end of file
+- [Teleport Kubernetes Access for CI/CD](./guides/cicd.mdx): Teleport Kubernetes
+  Access for CI/CD.
+- [Teleport Kubernetes Access and Trusted Clusters](./guides/federation.mdx):
+  Federated Access using Teleport Trusted Clusters.
+- [Kubernetes Access Multiple Clusters](./guides/multiple-clusters.mdx):
+  Connecting Multiple Clusters with Teleport Kubernetes Access.
+- [Kubernetes Access from a Standalone Teleport
+  Cluster](./guides/standalone-teleport.mdx): Connecting a Teleport cluster to a
```

Comment:
Not part of your change set, but something I noticed; these two link descriptions use a different tense than the others. What about:

```suggestion
  Connect Multiple Clusters with Teleport Kubernetes Access.
- [Kubernetes Access from a Standalone Teleport
  Cluster](./guides/standalone-teleport.mdx): Connect a Teleport cluster to a
```